src/main/java/org/folio/oaipmh/mappers/MarcXmlMapper.java

```diff
@@ -33,11 +36,20 @@ public Node convert(String source) {
         marcXmlWriter.write(record);
       }
       marcXmlWriter.close();
-      return domResult.getNode();
+      return postProcess(domResult.getNode().getFirstChild().getFirstChild());
     } catch (IOException e) {
       throw new UncheckedIOException(e); //should never happen
     }
+  }
 
+  /**
+   * Empty post-processing.
+   *
+   * @param source Node DOM representation of MarcXml result.
+   * @return same Node
+   */
+  public Node postProcess(Node source) {
```

Comment:
Let's get rid of this method since it is useless for this class.

Reply:
Done